### PR TITLE
render: switch to kube-core file layout

### PR DESCRIPTION
**Pull request by sttts**
Goal: be a drop-in for kube-core in the installer.

### CODE REVIEW

**Comment by sttts** [comment]
/hold

until smoke test with cluster-up is green

File: manifests/bootkube/manifests/kube-apiserver-daemonset.yaml [hunk 1]
```diff
@@ -24,6 +24,7 @@ spec:
         k8s-app: "kube-apiserver"
         openshift.io/control-plane: "true"
         openshift.io/component: "api"
+        checkpointer.alpha.coreos.com/checkpoint: "true"
```
Comment (mfojtik) [inline manifests/bootkube/manifests/kube-apiserver-daemonset.yaml]: do we want to checkpoint these? i thought we want to checkpoint only operator created resources

/cc @deads2k

**Comment by mfojtik** [comment]
/hold

@sttts can you bump library-go and replace the template?

one question about checkpointer otherwise LGTM

File: manifests/bootkube/manifests/kube-apiserver-daemonset.yaml (hunk 1 above)
Comment (sttts) [reply manifests/bootkube/manifests/kube-apiserver-daemonset.yaml]: Makes sense. removing.

**Comment by sttts** [comment]
/hold cancel

Moved to asset lib to library-go and remove the checkpointing annotation.

**Pull request merged**